spec/sample_reports/blastn_sample/job.yaml

```diff
@@ -0,0 +1,8 @@
+--- !ruby/object:SequenceServer::BLAST::Job
+id: blastn_sample
+method: blastn
+
+databases:
+- !ruby/struct:SequenceServer::Database
+  name: "../database/sample/transcripts/Solenopsis_invicta/Sinvicta2-2-3.cdna.subset.fasta"
```

Comment:
Even if it doesn't make or break anything, it should be absolute path so as to be representative of how SS works.